### PR TITLE
Remove support for IE < 11 (fixes #1863)

### LINKED ISSUE #1863: Legacy Internet Explorer Support?
With [support ended for Internet Explorer 10 ](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-10-end-of-support) I thought I'd open an official issue on our support for old Internet Explorer versions.

It's not a huge issue, but at some point it would be nice to sweep out all the code we have for old IE support. Even if we still support 10 it would be a nice cleanup, and not a huge task.

Comments?

Note: Even in my latest MouseTracker patch (in progress, coming v2.5.0) I'm still testing back to IE 9 (via emulation on IE 11)...I'd go back to 8 but we fail there - could be my test page or OpenSeadragon, but that debugging is a project for after this patch gets in...

### PR DESCRIPTION
**Pull request by msalsbery**

### CODE REVIEW

**Comment by msalsbery** [comment]
@iangilman  I fixed the navigator tests failing issue along with this PR...I can separate that fix from this "Remove support for IE < 11" PR to another PR if you'd prefer

File: changelog.txt [hunk 1]
```diff
@@ -33,6 +33,9 @@ OPENSEADRAGON CHANGELOG
 * Viewer: added a canvas-contextmenu event (#1872 @msalsbery)
 * Added additional documentation for the zoomPerSecond viewer option (#1872 @msalsbery)
 * MouseTracker: Per #1863, dropped support for Internet Explorer < 11 (#1872 @msalsbery)
+* Dropped support for older browsers (IE < 11) (#1949 @msalsbery)
```
Comment (iangilman) [inline changelog.txt:36]: I think this and the item below should be listed as breaking changes (they should be prefixed by "BREAKING CHANGE" and put at the top above the deprecation). They are not breaking changes the we expect to affect anyone, but it's probably still worth calling them out.

**Review by iangilman — CHANGES_REQUESTED** [review]
Looks great! I think it's fine to keep the test fix in here.

Just a little comment on the change log…

File: changelog.txt (hunk 1 above)
Comment (msalsbery) [reply changelog.txt]: > I think this and the item below should be listed as breaking changes (they should be prefixed by "BREAKING CHANGE" and put at the top above the deprecation). They are not breaking changes the we expect to affect anyone, but it's probably still worth calling them out.

Done

**Review by iangilman — APPROVED** [review]
Awesome, thank you!

**Pull request merged**